public/assets/css/bookmarklet.css

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2015 UC Berkeley (UCB) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+body {
+  background-color: inherit;            /* Ensure that the parent page remains visible behind the iFrame */
+}
+
+/* Boostrap overrides */
+
+.modal-backdrop {
+  position: fixed;                      /* Ensure that the backdrop remains visible when scrolling the parent page */
+  bottom: 0;                            /* Ensure that the backdrop stretches until the bottom of the page */
+}
+
+.modal-header {
+  border-bottom: none;
+}
+
+.modal-body {
+  padding-top: 0;
+}
+
+.modal-body h5:first-child {
+  margin-top: 0;
+}
+
+.modal-footer {
+  border-top: none;
+  padding: 0;
+}
+
+/* Overview panel */
+
+.collabosphere-overview-options-container {
+  padding: 0 15px 10px;
+}
+
+/* Select items from page */
+
+li.collabosphere-item-container {
+  padding: 10px 10px;
```

Comment:
@nicolaasmatthijs you might want to combine this with a shorthand `padding: 10px`. I'm also wondering whether we should add any linting to the CSS: https://github.com/ets-berkeley-edu/calcentral/blob/master/.scss-lint.yml